### PR TITLE
HKG: add sorento hev 4th gen camera fw

### PR DESCRIPTION
**Pull request by crwusiz**
dongle id : 75ced1efe488a089

### CODE REVIEW

File: selfdrive/car/hyundai/fingerprints.py [hunk 1]
```diff
@@ -1653,6 +1653,8 @@
       b'\xf1\x00MQ4HMFC  AT KOR LHD 1.00 1.12 99210-P2000 230331',
       b'\xf1\x00MQ4HMFC  AT USA LHD 1.00 1.10 99210-P2000 210406',
       b'\xf1\x00MQ4HMFC  AT USA LHD 1.00 1.11 99210-P2000 211217',
+      b'\xf1\x00MQ4HMFC  AT USA LHD 1.00 1.13 99210-P2000 230619',
+      b'\xf1\x00MQ4HMFC  AT KOR LHD 1.01 1.03 99210-P2500 230914',
```
Comment (sshane) [inline selfdrive/car/hyundai/fingerprints.py:1657]: Why are there two dates?

Comment (crwusiz) [reply selfdrive/car/hyundai/fingerprints.py:1657]: usa fw is -> Add Fingerprint for 2024 Kia Sorento Hybrid #30683

Comment (sshane) [reply selfdrive/car/hyundai/fingerprints.py:1657]: Why would you add it here?

**Comment by sshane** [comment]
Can you drive a bit more so we can see what your lateral accel factor is?

**Comment by crwusiz** [comment]
> Can you drive a bit more so we can see what your lateral accel factor is?

It's someone else's vehicle, so we haven't checked any further information yet, we've only checked some information after driving in the device pairing and dashcam status.

The other camera firmware is the same vehicle (USA) and it is not applied to the master wrench's fingerprint, so I added it together.

**Comment by sshane** [comment]
There was a lack of torque on that USA car, but you have a different camera part number, so interested in seeing what results you get.

**Comment by sshane** [comment]
Let me know when it's good for me to check

**Comment by sshane** [comment]
Closing since this user is no longer actively driving.

**Pull request closed without merging**